bodhi/services/updates.py

```diff
@@ -213,6 +213,10 @@ def query_updates(request):
         query = query.join(Update.builds).join(Build.package)
         query = query.filter(or_(*[Package.name==pkg for pkg in packages]))
 
+    package = None
+    if packages and len(packages):
```

Comment:
Isn't this a little redundant since if ``packages`` is a ``[]`` ``if packages`` will return ``False``?